exercises/practice/word-count/.meta/config.json

```diff
@@ -1,6 +1,13 @@
 {
   "blurb": "Given a phrase, count the occurrences of each word in that phrase.",
   "authors": [],
+  "contributors": [
+    "canweriotnow",
+    "npostavs",
```

Comment:
I made a minor fix to the test code (see #126); but I didn't really contribute to the exercise itself.

Reply:
Ditto! ☝🏽